packages/stardog-language-utils/src/namespaceUtils.ts

```diff
@@ -21,6 +21,11 @@ const abbreviate = (
   return newIri;
 };
 
+export const splitNamespace = (namespace: string) => {
+  const [alias, ...splitPrefix] = namespace.split('=');
```

Comment:
simpler to use the `limit` 2nd param to `split`?

Reply:
limit does not return the leftover text

Reply:
weird

Reply:
```
'1=2=3='.split('=',2)
>[ '1', '2' ]
```